gym/envs/safety/semisuper.py

```diff
@@ -15,13 +15,10 @@
 class SemisuperEnv(gym.Env):
     def step(self, action):
         assert self.action_space.contains(action)
-        self.monitor._before_step(action)
```

Comment:
I have no idea why this env was doing monitor stuff here. I'm assuming it was a mistake.